tests/Behat/UserGraphQLContext/UserGraphQLContext.php

```
@@ -199,65 +213,49 @@
      * @Then graphql response should be null
      */
     public function queryResponseShouldBeNull(): void
     {
-        $userData = json_decode(
-            $this->response->getContent(),
-            true
-        )['data'][$this->queryName];
+        $content = $this->getPageContent();
+        $userData = json_decode($content, true)['data'][$this->queryName];
 
         Assert::assertNull($userData);
```

PHPMD (2.15.0)

Avoid using static access to class '\PHPUnit\Framework\Assert' in method 'queryResponseShouldBeNull'. (Clean Code Rules)

(StaticAccess)